lib/mongoCrudHandler.js

```diff
@@ -87,13 +87,13 @@ module.exports = function init(config) {
     db.seagull.insert({_id: pair.id, value: crypted}, {w: 1}, function (err, result) {
       if (err) {
         if (err.code == 11000) {    // this is a mongo error for duplicate key
-          done({statuscode: 400, message: 'duplicate key', detail: err}, null);
+          done({statusCode: 400, message: 'duplicate key', detail: err}, null);
```

Comment:
these `statusCode` are internal aren't they? i.e. nothing is going to be expecting  `statuscode` at all?